core/site_scons/models/D002/discovery2.py

```diff
@@ -76,25 +69,20 @@ def configure(
 
     defines += [
         "USE_DMA2D",
-        "FRAMEBUFFER",
         "FRAMEBUFFER32BIT",
```

Comment:
not needed anymore

Reply:
or rather, we need FRAMEBUFFER instead of FRAMEBUFFER32BIT

Reply:
I removed FRAMEBUFFER32BIT. FRAMEBUFFER is defined few lines below. See 0b1c85d7a7e21a583c521d8061a32f0b1901a0ca